open-metadata-resources/open-metadata-deployment/README.md

```diff
@@ -3,8 +3,6 @@
 
```

Comment:
This file probably needs rewriting (can be in a separate issue!). Much of this content is fine in the new helm/vdc tree on our samples repo.

This doesn't really add value here and I would probably replace with a simple description of the subdirectories
I would also note that additional samples are now to be found in the new repo
The actual charts directory probably then needs an improved readme which refers to the existing tutorial information for setting up the lab chart (as that is all that is left) - similar to what we have in the compose/tutorials directory

As above - happy if you want to give this a go, or otherwise ok to merge and raise a new issue which I can address